<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitPushDialog</class>
 <widget class="QDialog" name="CommitPushDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commit/Push</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelCurrentBranchName_2">
       <property name="text">
        <string>Remote:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxRemote"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Branch</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonCurrentBranch">
        <property name="text">
         <string>Current branch</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButtonExistingBranch">
        <property name="text">
         <string>Existing branch</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxBranch">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButtonNewBranch">
        <property name="text">
         <string>New branch</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditNewBranchName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelCurrentBranchName">
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxIncludeStatus">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Include the output of &amp;quot;git status&amp;quot; in the commit message template when using an editor to prepare the commit message. &lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---status&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;More information&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Include status to the message</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxForce">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Force</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAmend">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Replace the tip of the current branch by creating a new commit.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---amend&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;More info&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Amend to previous commit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Message:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true"/>
       </item>
       <item>
        <widget class="CompletionTextEdit" name="textEditMessage">
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use the given text as the commit message. &lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---messageltmsggt&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;More information&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Files:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonAddAll">
           <property name="text">
            <string>All</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonAddIndexed">
           <property name="text">
            <string>Indexed</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonAddNone">
           <property name="text">
            <string>None</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCommit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPush">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Push</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompletionTextEdit</class>
   <extends>QTextEdit</extends>
   <header>widgets/completiontextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioButtonCurrentBranch</tabstop>
  <tabstop>radioButtonExistingBranch</tabstop>
  <tabstop>comboBoxBranch</tabstop>
  <tabstop>radioButtonNewBranch</tabstop>
  <tabstop>lineEditNewBranchName</tabstop>
  <tabstop>checkBoxIncludeStatus</tabstop>
  <tabstop>checkBoxForce</tabstop>
  <tabstop>checkBoxAmend</tabstop>
  <tabstop>textEditMessage</tabstop>
  <tabstop>toolButtonAddAll</tabstop>
  <tabstop>toolButtonAddIndexed</tabstop>
  <tabstop>toolButtonAddNone</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButtonCommit</tabstop>
  <tabstop>pushButtonPush</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonExistingBranch</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxBranch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNewBranch</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditNewBranchName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonCurrentBranch</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelCurrentBranchName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAmend</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEditMessage</sender>
   <signal>textChanged()</signal>
   <receiver>CommitPushDialog</receiver>
   <slot>checkButtonsEnable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAmend</sender>
   <signal>toggled(bool)</signal>
   <receiver>CommitPushDialog</receiver>
   <slot>checkButtonsEnable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkButtonsEnable()</slot>
 </slots>
</ui>
